Python/sqlmlutils/packagemanagement/pipdownloader.py

```diff
@@ -12,11 +12,12 @@
 
 class PipDownloader:
 
-    def __init__(self, connection: ConnectionInfo, downloaddir: str, targetpackage: str):

```

Comment:
Is there a reason to remove the default value ?

Reply:
Is this comment in the wrong place? which default value are you asking about?

---
In reply to: [502129225](https://github.com/microsoft/sqlmlutils/pull/82#discussion_r502129225) [](ancestors = 502129225)

Reply:
Anyway, we only want default values for language_name in the top level user-facing functions. Any functions or classes that are private we do NOT want to have default values, because we want to make sure we are passing the values from the front to all the private functions.

---
In reply to: [502636207](https://github.com/microsoft/sqlmlutils/pull/82#discussion_r502636207) [](ancestors = 502636207,502129225)